tests/system/fim/common_tasks/verify_alerts.yml

```diff
@@ -0,0 +1,15 @@
+---  
+  - name: "Verify alert log"
+    script: "verify_alerts_json.py -f {{ file_list_path }} -e {{ syscheck_event }}"
```

Comment:
Hey @DFolchA thank you for adding the previously requested changes. I'm wondering where these `file_list_path` and `syscheck_event` variables are coming from. I assume that the `private_ip` comes from the generated Ansible hostfile, but not these two.

Reply:
Thes two variables should be provided to the tasks by the playbook calling them.
`file_list_path` is the path to the file containing all the files created/modified it should be created when creating the files and provided to the machines that need it.
`syscheck_event` is the kind of alert event we are looking for: added, modified or deleted; this is one of the parameters of the test.